examples/docker/docker/Dockerfile

```diff
@@ -3,7 +3,8 @@ FROM ubuntu:20.04
 # Install OS-level libraries.
 RUN apt-get update -y && DEBIAN_FRONTEND="noninteractive" apt-get install -y \
     python3 \
-    python3-pip && \
+    python3-pip \
+    graphviz-dev && \
```

Comment:
@rbetz @mtandon-insitro Do you two know why @annashch-insitro might have needed to install `graphviz-dev` here? I thought your way of depending on graphviz through `extras` meant most users wouldn't need to install `graphviz-dev`?

Reply:
redun.cli imports redun.visualize which imports pygraphviz. I fixed this in a recent PR to disable the visualization if pygraphviz isn't importable, so if we do a release this PR shouldn't be needed

Reply:
Ah, @annashch-insitro this is fixed in internal redun. I just need to push it to this public repo.

Reply:
Sounds good, thanks for clarifying, I'll go ahead and close the PR then.